src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/InitialPositionInStreamExtended.java

```diff
@@ -20,7 +20,7 @@
  * Class that houses the entities needed to specify the position in the stream from where a new application should
  * start.
  */
-class InitialPositionInStreamExtended {
+public class InitialPositionInStreamExtended {
```

Comment:
Does this need to be changed to `public`?

Reply:
This is required in the DDB streams adapter and hence it had to be made public